### PR TITLE
Update simple.md

### LINKED ISSUE #26332: Misleading statement about Authorize/AllowAnoynmous override
The document in its current form leads one to believe that combining Authorize with AllowAnonymous on a controller will lead to Authorize being ignored, this is only partially true.

What really happens is that the result of the authorization is ignored it is, however, still performed.

This behavior means that the authentication pipeline will indeed run and you will have access to the User Identity in the HttpContext _if_ the user was authenticated. 

This can be seen as an alternative to calling HttpContext.AuthenticateAsync() within the controller if you have an endpoint that switches on the authentication state anyway.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4c3930e5-8a4f-8bdf-7167-9aed8afc4253
* Version Independent ID: cea5942d-fb01-f5bf-3b63-25873d5b79cf
* Content: [Simple authorization in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/authorization/simple?view=aspnetcore-6.0)
* Content Source: [aspnetcore/security/authorization/simple.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/security/authorization/simple.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

---
[Associated WorkItem - 33226](https://dev.azure.com/msft-skilling/Content/_workitems/edit/33226)

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Fixes #26332

@Blackclaws please review

[Internal review url](https://review.learn.microsoft.com/en-us/aspnet/core/security/authorization/simple?view=aspnetcore-8.0&branch=pr-en-us-27804)

image

### CODE REVIEW

File: aspnetcore/security/authorization/simple.md [hunk 1]
```diff
@@ -66,7 +66,9 @@ public class AccountController : Controller
 This would allow only authenticated users to the `AccountController`, except for the `Login` action, which is accessible by everyone, regardless of their authenticated or unauthenticated / anonymous status.
 
 > [!WARNING]
-> `[AllowAnonymous]` bypasses all authorization statements. If you combine `[AllowAnonymous]` and any `[Authorize]` attribute, the `[Authorize]` attributes are ignored. For example if you apply `[AllowAnonymous]` at the controller level, any `[Authorize]` attributes on the same controller (or on any action within it) are ignored.
+> `[AllowAnonymous]` bypasses authorization statements. If you combine `[AllowAnonymous]` and an `[Authorize]` attribute, the `[Authorize]` attributes are ignored. For example if you apply `[AllowAnonymous]` at the controller level:
+> * Any `[Authorize]` attributes on the same controller or action methods on the controller, the authorization requirements specified by `[Authorize]` ware ignored.
+> * Any related authentication is still be done.
```
Comment (Blackclaws) [inline aspnetcore/security/authorization/simple.md:71]: Would change to: Authentication middleware is not short-circuited but doesn't need to succeed.

File: aspnetcore/security/authorization/simple.md [hunk 2]
```diff
@@ -66,7 +66,9 @@ public class AccountController : Controller
 This would allow only authenticated users to the `AccountController`, except for the `Login` action, which is accessible by everyone, regardless of their authenticated or unauthenticated / anonymous status.
 
 > [!WARNING]
-> `[AllowAnonymous]` bypasses all authorization statements. If you combine `[AllowAnonymous]` and any `[Authorize]` attribute, the `[Authorize]` attributes are ignored. For example if you apply `[AllowAnonymous]` at the controller level, any `[Authorize]` attributes on the same controller (or on any action within it) are ignored.
+> `[AllowAnonymous]` bypasses authorization statements. If you combine `[AllowAnonymous]` and an `[Authorize]` attribute, the `[Authorize]` attributes are ignored. For example if you apply `[AllowAnonymous]` at the controller level:
+> * Any `[Authorize]` attributes on the same controller or action methods on the controller, the authorization requirements specified by `[Authorize]` ware ignored.
```
Comment (Blackclaws) [inline aspnetcore/security/authorization/simple.md:70]: Any authorization requirements from `[Authorize]` attributes on the same controller or action methods on the controller are ignored.

**Review by Blackclaws — CHANGES_REQUESTED** [review]
I've tried to make the first statement a bit more concise and expanded the second statement a bit because it didn't exactly fit what's happening.

File: aspnetcore/security/authorization/simple.md (hunk 2 above)
Comment (Rick-Anderson) [inline aspnetcore/security/authorization/simple.md:70]:
```suggestion
> * Any authorization requirements from `[Authorize]` attributes on the same controller or action methods on the controller are ignored.
```

File: aspnetcore/security/authorization/simple.md [hunk 3]
```diff
@@ -66,7 +66,9 @@ public class AccountController : Controller
 This would allow only authenticated users to the `AccountController`, except for the `Login` action, which is accessible by everyone, regardless of their authenticated or unauthenticated / anonymous status.
 
 > [!WARNING]
-> `[AllowAnonymous]` bypasses all authorization statements. If you combine `[AllowAnonymous]` and any `[Authorize]` attribute, the `[Authorize]` attributes are ignored. For example if you apply `[AllowAnonymous]` at the controller level, any `[Authorize]` attributes on the same controller (or on any action within it) are ignored.
+> `[AllowAnonymous]` bypasses authorization statements. If you combine `[AllowAnonymous]` and an `[Authorize]` attribute, the `[Authorize]` attributes are ignored. For example if you apply `[AllowAnonymous]` at the controller level:
+> * Any authorization requirements from `[Authorize]` attributes on the same controller or action methods on the controller are ignored.
+> * Any related authentication is still be done.
```
Comment (Rick-Anderson) [inline aspnetcore/security/authorization/simple.md:71]:
```suggestion
> * Authentication middleware is not short-circuited but doesn't need to succeed.
```

**Comment by Rick-Anderson** [comment]
@Blackclaws can you review the published doc. If you'd like changes, can you PR it? 
At the top right of the article, select the Edit pen icon:

image

Select the pen icon again. Save.

**Pull request merged**